test_chesapeakecvpr_models.py

```diff
@@ -47,6 +53,14 @@ def set_up_parser() -> argparse.ArgumentParser:
         help="path to the CSV file to write results",
         metavar="FILE",
     )
+    parser.add_argument(
+        "-d",
+        "--device",
+        default=0,
+        type=int,
+        help="GPU ID to use, ignored if no GPUs are available",
```

Comment:
It doesn't have to be a GPU ID, it could also be a CPU ID. PyTorch supports both.

Reply:
See where `args.device` is used in the script -- it isn't in `torch.device(...)` it is passed to the gpus param of pl.Trainer

Reply:
Also, I know pytorch allows `torch.device("cpu", 0)` but I don't actually know what that does... This is what I was saying I tested the other day, even if you set it to 0 or 1, it still runs on all CPU cores. Maybe there is a difference if your machine actually has multiple CPUs?

Reply:
Ah, I see. Yeah, it's only for when you have multiple CPUs, it doesn't specify the specific core to run on.